tests/ignite/metrics/test_epoch_metric.py

```diff
@@ -113,7 +114,7 @@ def compute_fn(y_preds, y_targets):
 
     em.reset()
     output1 = (torch.rand(4, 3), torch.randint(0, 2, size=(4, 4), dtype=torch.long))
-    with pytest.warns(RuntimeWarning):
+    with pytest.warns(EpochMetricWarning, match=r"Probably, there can be a problem with `compute_fn`"):
```

Comment:
Just a question: Is there a reason to use a raw string here but not in the actual implementation?

Reply:
This can be helpful if we would like to use it with a regexp.